src/Microsoft.AspNet.SignalR.Core/Configuration/IConfigurationManager.cs

```diff
@@ -52,10 +52,16 @@ public interface IConfigurationManager
 
         /// <summary>
         /// Gets or sets a <see cref="TimeSpan"/> representing tell the client to wait before restablishing a
-        /// long poll connection after data is sent from the server. 
+        /// long poll connection after data is sent from the server.
         /// The default value is 0.
         /// </summary>
         [SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "long", Justification = "Longpolling is a well known term")]
         TimeSpan LongPollDelay { get; set; }
+
+        /// <summary>
+        /// Gets or sets the maximum number of scaleout mappings per scaleout stream stored on the server.
+        /// The default value is 1000000
+        /// </summary>
+        int? MaxScaleoutMappingsPerStream { get; set; }
```

Comment:
@halter73 - actually one more reason to use nullable is that I need to add a number of new ctors taking this setting. Since the types are public I keep the old ctors for backwards compatibility and pass null. I initialize to the default value only in one place. If the setting on the IConfigurationManager was not nullable I would have to initialize it to 1000000 in at least two places (DefaultConfigurationManager and ScaleoutMappingStore). In this case I would still keep the nullable parameter on ScaleoutStreamManager ctor. If you wanted to make this param non-nullable then I would have to initialize to the default value there as well. Nullable allows me to initialize only in one place.